src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/_addresses.html.twig

```diff
@@ -1,14 +1,18 @@
-<h4 class="ui top attached styled header">
-    {{ 'sylius.ui.shipping_address'|trans }}
-</h4>
-<div class="ui attached segment" id="shipping-address">
-    {% include '@SyliusAdmin/Common/_address.html.twig' with {'address': order.shippingAddress} %}
-</div>
-<h4 class="ui attached styled header">
+{% if order.shippingAddress is not null %}
```

Comment:
Hiding whole block about the shipping address is useful for shops selling digital products only, as they don't ask customer for a shipping address and admins don't want to see it also.

The initial idea was to hide the whole block about the billing address also, but there is an edit button. Any ideas what to do with it? Move to the order actions next to "Cancel" button maybe?